indexer_util/indexer_util/indexer_util.py

```diff
@@ -76,10 +76,11 @@ def _wait_elasticsearch_healthy(es):
     logging.getLogger("elasticsearch").setLevel(logging.INFO)
 
 
-def init_elasticsearch(elasticsearch_url, index_name):
-    """Performs Elasticsearch initialization.
+def maybe_create_elasticsearch_index(elasticsearch_url, index_name):
+    """Maybe create Elasticsearch index.
```

Comment:
IMO the named helper method makes this code clear enough to be self-documenting outside of the args, otherwise I think "Wait for Elasticsearch to become available and create the index if it doesn't already exist" would suffice here